crates/ra_cfg/src/lib.rs

```diff
@@ -35,11 +35,20 @@ impl CfgOptions {
         self.check(&parse_cfg(attr))
     }
 
-    pub fn insert_atom(&mut self, key: SmolStr) {
-        self.atoms.insert(key);
+    pub fn insert(&mut self, raw_string: &str) {
+        match raw_string.find('=') {
+            None => {
+                self.atoms.insert(raw_string.into());
+            }
+            Some(pos) => {
+                let key = &raw_string[..pos];
+                let value = raw_string[pos + 1..].trim_matches('"');
+                self.key_values.insert((key.into(), value.into()));
+            }
+        }
     }
 
-    pub fn remove_atom(&mut self, name: &str) {
+    pub fn remove(&mut self, name: &str) {
```

Comment:
I renamed this for consistency, but this fn hasn't been used anywhere in a while, and maybe should be removed?